<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>570</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon-resources.qrc">
    <normaloff>:/icons/icons/sonicawe.png</normaloff>:/icons/icons/sonicawe.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1056</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionToggleTopFilterWindow"/>
    <addaction name="actionToggleOperationsWindow"/>
    <addaction name="actionToggleHistoryWindow"/>
    <addaction name="actionToggleTimelineWindow"/>
    <addaction name="separator"/>
    <addaction name="actionToggleToolToolBox"/>
    <addaction name="actionToggleCreationToolBox"/>
    <addaction name="actionToggleTransformToolBox"/>
    <addaction name="actionToggleNavigationToolBox"/>
    <addaction name="actionToggleTimeControlToolBox"/>
    <addaction name="actionToggleSelectionToolBox"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_recording"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_project"/>
    <addaction name="separator"/>
    <addaction name="actionImport_audio"/>
    <addaction name="actionExport_audio"/>
    <addaction name="actionExport_selection"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindows"/>
  </widget>
  <widget class="QToolBar" name="toolToolBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionActivateNavigation"/>
   <addaction name="actionAmplitudeBrush"/>
   <addaction name="actionAirbrush"/>
   <addaction name="actionSmoothBrush"/>
   <addaction name="separator"/>
   <addaction name="actionToggle_piano_grid"/>
   <addaction name="actionToggle_hz_grid"/>
   <addaction name="separator"/>
   <addaction name="actionPlaySelection"/>
   <addaction name="actionFollowPlayMarker"/>
   <addaction name="separator"/>
   <addaction name="actionMatlabOperation"/>
   <addaction name="actionMatlabFilter"/>
   <addaction name="actionRecord"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="topFilterWindow">
   <property name="minimumSize">
    <size>
     <width>215</width>
     <height>209</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>524287</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Top filter</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="layerWindowContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="layerWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="deleteFilterButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="operationsWindow">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Operations</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="operationsTree">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="historyWindow">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>History</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QToolBar" name="selectionToolBox">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Selection Tool Box</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionActivateSelection"/>
   <addaction name="actionSquareSelection"/>
   <addaction name="actionSplineSelection"/>
   <addaction name="actionPolygonSelection"/>
   <addaction name="actionPeakSelection"/>
   <addaction name="actionCutoffSelection"/>
   <addaction name="actionActionAdd_selection"/>
   <addaction name="actionActionRemove_selection"/>
   <addaction name="separator"/>
   <addaction name="actionCropSelection"/>
   <addaction name="actionMoveSelection"/>
   <addaction name="actionMoveSelectionTime"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetTimeline">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="windowTitle">
    <string>Timeline</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <action name="actionActivateSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/icon-circle-selection.png</normaloff>
     <normalon>:/icons/icons/icon_select.png</normalon>:/icons/icons/icon-circle-selection.png</iconset>
   </property>
   <property name="text">
    <string>Activate Selection</string>
   </property>
   <property name="toolTip">
    <string>Activate selection [S]</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionActivateNavigation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/icon-navigation.png</normaloff>
     <normalon>:/icons/icons/icon_move.png</normalon>:/icons/icons/icon-navigation.png</iconset>
   </property>
   <property name="text">
    <string>Activate Navigation</string>
   </property>
   <property name="toolTip">
    <string>Activate navigation [N]</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionToggleTopFilterWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Top filter</string>
   </property>
   <property name="toolTip">
    <string>Toggle the top filter window</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionToggle_piano_grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/icon-toggle-piano.png</normaloff>:/icons/icons/icon-toggle-piano.png</iconset>
   </property>
   <property name="text">
    <string>Toggle piano grid</string>
   </property>
   <property name="toolTip">
    <string>Toggle piano grid</string>
   </property>
  </action>
  <action name="actionPlaySelection">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/icon-play.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Play Selection</string>
   </property>
   <property name="toolTip">
    <string>Play the selected area [Space]</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionToggleSelectionToolBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Selection tool box</string>
   </property>
   <property name="toolTip">
    <string>Toggle the selection tool box</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionToggle_hz_grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/icon-toggle-hz.png</normaloff>:/icons/icons/icon-toggle-hz.png</iconset>
   </property>
   <property name="text">
    <string>Toggle frequency markers</string>
   </property>
   <property name="toolTip">
    <string>Toggle frequency markers</string>
   </property>
  </action>
  <action name="actionActionAdd_selection">
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/icon-save-selection.png</normaloff>:/icons/icons/icon-save-selection.png</iconset>
   </property>
   <property name="text">
    <string>actionAdd_selection</string>
   </property>
   <property name="toolTip">
    <string>Remember the selection</string>
   </property>
  </action>
  <action name="actionActionRemove_selection">
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/icon-silence-selection.png</normaloff>:/icons/icons/icon-silence-selection.png</iconset>
   </property>
   <property name="text">
    <string>actionRemove_selection</string>
   </property>
   <property name="toolTip">
    <string>Silences the selected region [Del]</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionCropSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/sonicawe.png</normaloff>
     <normalon>:/icons/icons/icon_move.png</normalon>:/icons/icons/sonicawe.png</iconset>
   </property>
   <property name="text">
    <string>Crop Selection</string>
   </property>
   <property name="toolTip">
    <string>Crop selection [C]</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMoveSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/sonicawe.png</normaloff>
     <normalon>:/icons/icons/icon_move.png</normalon>:/icons/icons/sonicawe.png</iconset>
   </property>
   <property name="text">
    <string>Move Selection</string>
   </property>
   <property name="toolTip">
    <string>Move selection [M]</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMoveSelectionTime">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/sonicawe.png</normaloff>
     <normalon>:/icons/icons/icon_move.png</normalon>:/icons/icons/sonicawe.png</iconset>
   </property>
   <property name="text">
    <string>Move Selection in Time</string>
   </property>
   <property name="toolTip">
    <string>Move selection in Time [T]</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMatlabFilter">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/sonicawe.png</normaloff>
     <normalon>:/icons/icons/icon_move.png</normalon>:/icons/icons/sonicawe.png</iconset>
   </property>
   <property name="text">
    <string>Matlab filter</string>
   </property>
   <property name="toolTip">
    <string>Apply Matlab filter to selection</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMatlabOperation">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/sonicawe.png</normaloff>
     <normalon>:/icons/icons/icon_move.png</normalon>:/icons/icons/sonicawe.png</iconset>
   </property>
   <property name="text">
    <string>Matlab operation</string>
   </property>
   <property name="toolTip">
    <string>Apply Matlab operation to everything</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/sonicawe.png</normaloff>
     <normalon>:/icons/icons/icon_move.png</normalon>:/icons/icons/sonicawe.png</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
   <property name="toolTip">
    <string>Toggle recording [R]</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFollowPlayMarker">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/sonicawe.png</normaloff>
     <normalon>:/icons/icons/icon-play.png</normalon>:/icons/icons/sonicawe.png</iconset>
   </property>
   <property name="text">
    <string>Follow play marker</string>
   </property>
   <property name="toolTip">
    <string>Follow the play marker when playing</string>
   </property>
  </action>
  <action name="actionToggleOperationsWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Operations</string>
   </property>
   <property name="toolTip">
    <string>Toggle the operations window</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="actionToggleToolToolBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tools</string>
   </property>
   <property name="toolTip">
    <string>Toggle the tool box</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionToggleCreationToolBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sound creation</string>
   </property>
   <property name="toolTip">
    <string>Toggle the sound creation tool box</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionToggleTransformToolBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Transform</string>
   </property>
   <property name="toolTip">
    <string>Toggle the transform tool box</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionToggleNavigationToolBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Navigation</string>
   </property>
   <property name="toolTip">
    <string>Toggle the navigation tool box</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionToggleTimeControlToolBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Time control</string>
   </property>
   <property name="toolTip">
    <string>Toggle the time control tool box</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionNew_recording">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New recording</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionImport_audio">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport_audio">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport_selection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export selection</string>
   </property>
  </action>
  <action name="actionToggleTimelineWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Timeline window</string>
   </property>
   <property name="toolTip">
    <string>Toggle the timeline window</string>
   </property>
  </action>
  <action name="actionToggleHistoryWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>History</string>
   </property>
   <property name="toolTip">
    <string>Toggle the history window</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionCutoffSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/icon-cutoff-selection.png</normaloff>:/icons/icons/icon-cutoff-selection.png</iconset>
   </property>
   <property name="text">
    <string>Cutoff selection</string>
   </property>
  </action>
  <action name="actionPeakSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/icon-peak-selection.png</normaloff>:/icons/icons/icon-peak-selection.png</iconset>
   </property>
   <property name="text">
    <string>Peak selection</string>
   </property>
   <property name="toolTip">
    <string>Peak selection</string>
   </property>
  </action>
  <action name="actionSquareSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/icon-square-selection.png</normaloff>:/icons/icons/icon-square-selection.png</iconset>
   </property>
   <property name="text">
    <string>Square selection</string>
   </property>
   <property name="toolTip">
    <string>Square selection</string>
   </property>
  </action>
  <action name="actionPolygonSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/icon-quad-selection.png</normaloff>:/icons/icons/icon-quad-selection.png</iconset>
   </property>
   <property name="text">
    <string>Polygon selection</string>
   </property>
   <property name="toolTip">
    <string>Polygon selection</string>
   </property>
  </action>
  <action name="actionSplineSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/icon-spline-selection.png</normaloff>:/icons/icons/icon-spline-selection.png</iconset>
   </property>
   <property name="text">
    <string>Spline selection</string>
   </property>
   <property name="toolTip">
    <string>Spline selection</string>
   </property>
  </action>
  <action name="actionAmplitudeBrush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/icon-amplitude-brush.png</normaloff>:/icons/icons/icon-amplitude-brush.png</iconset>
   </property>
   <property name="text">
    <string>Amplitude brush</string>
   </property>
   <property name="toolTip">
    <string>Amplitude brush</string>
   </property>
  </action>
  <action name="actionAirbrush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/icon-airbrush.png</normaloff>:/icons/icons/icon-airbrush.png</iconset>
   </property>
   <property name="text">
    <string>Airbrush</string>
   </property>
   <property name="toolTip">
    <string>Airbrush</string>
   </property>
  </action>
  <action name="actionSmoothBrush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon-resources.qrc">
     <normaloff>:/icons/icons/icon-smooth-brush.png</normaloff>:/icons/icons/icon-smooth-brush.png</iconset>
   </property>
   <property name="text">
    <string>Smooth brush</string>
   </property>
   <property name="toolTip">
    <string>Smooth brush</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon-resources.qrc"/>
 </resources>
 <connections/>
</ui>
